python-threatexchange/threatexchange/signal_type/md5.py

```diff
@@ -29,7 +29,7 @@ class VideoMD5Signal(signal_base.SimpleSignalType, signal_base.FileHasher):
     TYPE_TAG = "media_type_video"
 
     @classmethod
-    def hash_file(cls, file: pathlib.Path) -> str:
+    def hash_from_file(cls, file: pathlib.Path) -> str:
```

Comment:
Why this change?

Reply:
in the current master `hash_file` is only referenced `hash_cmd.py`everywhere else looks like it expects `hash_from_file` see https://github.com/facebook/ThreatExchange/blob/5acf1bf5fdb47873c83f71bb785b14f3583805c2/python-threatexchange/threatexchange/signal_type/signal_base.py#L177

Reply:
Whoops, that's definitely a refactor fail by me then.